test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,41 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env helps the user to set environmental variables which are needed force
+the usage of Minishift with docker command on VM host.
+INFO: This feature runs against persistent shell instance. To use non-default shell, please
+select one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_BASH parameter of make integration.
+
+  Scenario: Docker client is not set up
+  INFO: This scenario starts interactive shell instance, which will be closed in the end of this feature.
+    Given user starts shell instance
+     When executing in shell command "docker info"
+     Then stdout of shell should not contain "Operating System: Minishift Boot2Docker ISO Version"
+      And stdout of shell should not contain "Name: minishift"
+
+  Scenario: Starting minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift has state "Running"
+
+  Scenario: Setting Docker client using docker-env command
+     When executing in shell command "minishift docker-env" succeeds
+      And executing in shell shell's stdout of previous command line by line
+     Then executing in shell command "docker info" succeeds
+      And stdout of shell should contain "OSType: linux"
+      And stdout of shell should contain "Name: minishift"
+
+  Scenario: Unsetting Docker client using --unset flag of docker-env command
+     When executing in shell command "minishift docker-env --unset" succeeds
+      And executing in shell shell's stdout of previous command line by line
+      And executing in shell command "docker info"
+     Then stdout of shell should not contain "Operating System: Minishift Boot2Docker ISO Version"
+      And stdout of shell should not contain "Name: minishift"
+
+  Scenario: Closing shell instance
+  INFO: Removes the interactive shell instance.
```

Comment:
I doubt `Given` or `When` is missing here.

Reply:
@budhrg this step is actually not a test, but a job which needs to be done here, but I agree it looks a little bit strange, I could move it into the scenario where Minishift instance is being deleted. WDYT?

Reply:
Sure